packages/ui-kit/src/helpers/formatLabels.ts

```diff
@@ -0,0 +1,14 @@
+export interface FormatLabelsOptions {
+  labels?: string[]
+  formatter?: (labels: string[]) => string[]
+}
+
+export function formatLabels(options: FormatLabelsOptions): string[] | undefined {
+  const { labels, formatter } = options
+
+  if (formatter && typeof formatter !== 'function') {
```

Comment:
I think, this If block seems to me unnecessary. since formatter is function or undefined.

Reply:
That's okay for Typescript but for customers using native Javascript they won't be able to validate the formatter must be a function, this if statements ensure that if the user provides something different to a function in the formatter a useful error is thrown